### PR TITLE
add schema for CRD controllers

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
Pass schema to CRD cntrolller. 

**Motivation**
https://github.com/kubernetes-sigs/kubebuilder/issues/986

**Local Test**
Screenshot 2019-09-30 at 16 29 56

### CODE REVIEW

File: pkg/scaffold/v2/controller.go [hunk 1]
```diff
@@ -82,6 +82,7 @@ import (
 type {{ .Resource.Kind }}Reconciler struct {
 	client.Client
 	Log logr.Logger
+    Scheme *runtime.Scheme
```
Comment (DirectXMan12) [inline pkg/scaffold/v2/controller.go]: spacing is off here.  Maybe a tabs vs spaces issue?

Comment (camilamacedo86) [reply pkg/scaffold/v2/controller.go]: Fix. thank you.
This diff of layout did not appears in the idea. Have any specific/setup config that is used by you folks?

**Comment by camilamacedo86** [comment]
Hi @DirectXMan12, 

It is updated with the golden script now.  👍

File: testdata/project-v2/main.go [hunk 2]
```diff
@@ -40,6 +41,7 @@ func init() {
 	_ = clientgoscheme.AddToScheme(scheme)
 
 	_ = crewv1.AddToScheme(scheme)
+	_ = v1.AddToScheme(scheme)
 	_ = corev1.AddToScheme(scheme)
```
Comment (camilamacedo86) [inline testdata/project-v2/main.go]: It has no relation, see that this update is missing in the master branch: https://github.com/kubernetes-sigs/kubebuilder/pull/1066

File: pkg/scaffold/v2/webhook/webhook.go [hunk 3]
```diff
@@ -80,6 +80,11 @@ func (a *Webhook) GetInput() (input.Input, error) {
 	return a.Input, nil
 }
 
+// Validate validates the values
+func (g *Webhook) Validate() error {
+	return g.Resource.Validate()
+}
```
Comment (DirectXMan12) [inline pkg/scaffold/v2/webhook/webhook.go]: ?? was this just missing before

**Comment by Adirio** [comment]
Bug: #1119

**Pull request merged**